### PR TITLE
version control collection of favicons

### PR DESCRIPTION
**Pull request by sparty02**
this is just a follow-up commit to include the favicon collection in version control (should we need any of the other copies)

### CODE REVIEW

**Comment by thejameskyle** [comment]
Don't these need `<link>` tags? https://developer.apple.com/library/ios/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html

**Comment by sparty02** [comment]
I didn't actually implement the full blown favicon link set... it's currently still just the one link tag:
```
<link rel="icon" href="{{ "/favicon.ico" | prepend: site.baseurl }}" type="image/x-icon">
```

This commit is really just to version control the favicon set that I generated (so that we can add those links later if we want to).

**Comment by thejameskyle** [comment]
Ah, well do you mind adding them now?

**Comment by sparty02** [comment]
Done....it's a doozie!

**Comment by thejameskyle** [comment]
ahaha, omg.

File: _includes/head.html [hunk 1]
```diff
@@ -13,6 +13,23 @@
...
 
   <link rel="icon" href="{{ "/favicon.ico" | prepend: site.baseurl }}" type="image/x-icon">
+  
+  <link rel="apple-touch-icon" sizes="57x57" href="{{ "/apple-touch-icon-57x57.png" | prepend: site.baseurl }}">
+  <link rel="apple-touch-icon" sizes="114x114" href="{{ "/apple-touch-icon-114x114.png" | prepend: site.baseurl }}">
+  <link rel="apple-touch-icon" sizes="72x72" href="{{ "/apple-touch-icon-72x72.png" | prepend: site.baseurl }}">
+  <link rel="apple-touch-icon" sizes="144x144" href="{{ "/apple-touch-icon-144x144.png" | prepend: site.baseurl }}">
+  <link rel="apple-touch-icon" sizes="60x60" href="{{ "/apple-touch-icon-60x60.png" | prepend: site.baseurl }}">
+  <link rel="apple-touch-icon" sizes="120x120" href="{{ "/apple-touch-icon-120x120.png" | prepend: site.baseurl }}">
+  <link rel="apple-touch-icon" sizes="76x76" href="{{ "/apple-touch-icon-76x76.png">
```
Comment (thejameskyle) [inline _includes/head.html]: @sparty02  whoops, missing a closing `}}`

Comment (sparty02) [inline _includes/head.html]: Son of a bee sting.... fixing now...

Comment (sparty02) [inline _includes/head.html]: @see #87

**Pull request merged**